### PR TITLE
Added httpRequest command

### PR DESCRIPTION
**Pull request by dqdquint**
Passed Post for method and sdkData to bdoy for HttpRequest

### CODE REVIEW

File: frontend/src/Plugin.js [hunk 1]
```diff
@@ -143,6 +144,8 @@ class ShopgateAnalytics extends SgTrackingPlugin {
         sdkData.meta = rawData.meta;
       }
 
+      const checkoutCompletedEvent = new HttpRequest();
+      checkoutCompletedEvent.setMethod('POST').setPayload(sdkData).dispatch();
```
Comment (Carsten89) [inline frontend/src/Plugin.js]: isn't the name of the event now missing here?
+ it should be a different name

File: frontend/src/Plugin.js [hunk 2]
```diff
@@ -143,6 +144,8 @@ class ShopgateAnalytics extends SgTrackingPlugin {
         sdkData.meta = rawData.meta;
       }
 
+      const checkoutCompletedEvent = new HttpRequest();
```
Comment (sznowicki) [inline frontend/src/Plugin.js]: Also the url is missing for the endpoint

File: frontend/src/Plugin.js (hunk 1 above)
Comment (DannyShopgate) [reply frontend/src/Plugin.js]: Updated name of the event. Must've got the name tongue twisted.

File: frontend/src/Plugin.js (hunk 2 above)
Comment (DannyShopgate) [reply frontend/src/Plugin.js]: Updated with the url

**Comment by DannyShopgate** [comment]
Ran into an issue with the sdkData object not serializing correctly in the HttpRequest command. Updating review with latest iteration.

screen shot 2018-12-05 at 3 45 45 pm

File: frontend/src/Plugin.js (hunk 1 above)
Comment (DannyShopgate) [reply frontend/src/Plugin.js]: Originally misunderstood. Fixed object structure and added content type function for HttpRequest command

File: frontend/src/Plugin.js (hunk 2 above)
Comment (DannyShopgate) [reply frontend/src/Plugin.js]: Modified used url to account for development or production

**Pull request closed without merging**